bfv/bfv_test.go

```diff
@@ -38,21 +41,27 @@ type testContext struct {
 
 func TestBFV(t *testing.T) {
 
-	var err error
-
-	var defaultParams = DefaultParams[PN12QP109 : PN12QP109+4] // the default test runs for ring degree N=2^12, 2^13, 2^14, 2^15
+	var defaultParams = DefaultParams // the default test runs for ring degree N=2^12, 2^13, 2^14, 2^15
 	if testing.Short() {
-		defaultParams = DefaultParams[PN12QP109 : PN12QP109+2] // the short test suite runs for ring degree N=2^12, 2^13
+		defaultParams = DefaultParams[:2] // the short test suite runs for ring degree N=2^12, 2^13
```

Comment:
Not the same value as in the benchmarks

Reply:
It is true that consistency at this level might help if we integrate benchmarks to our CI workflows, but this might not a be priority.